src/kibana/plugins/discover/partials/save_search.html

```diff
@@ -2,11 +2,11 @@
   <div class="row">
     <div class="col-md-12">
       <div class="form-group">
-        <label class="control-label">Save Search</label>
-        <input ng-model="opts.savedSearch.title" input-focus class="form-control" placeholder="Name this search...">
+        <label for="SaveSearch" class="control-label">Save Search</label>
+        <input id="SaveSearch" ng-model="opts.savedSearch.title" input-focus class="form-control" placeholder="Name this search...">
       </div>
       <div class="form-group">
-        <button ng-click="opts.saveDataSource()" ng-disabled="!opts.savedSearch.title" type="submit" class="btn btn-primary">
+        <button ng-click="opts.saveDataSource()" aria-label="Save" ng-disabled="!opts.savedSearch.title" type="submit" class="btn btn-primary">
```

Comment:
Do we need `[aria-label]` on things that have just one child that is a text node with the same info?